Makefile

```diff
@@ -137,7 +137,13 @@ _output/share/lima/lima-guestagent.Linux-riscv64:
 .PHONY: manpages
 manpages: _output/bin/limactl$(exe)
 	@mkdir -p _output/share/man/man1
-	$< generate-man _output/share/man/man1 \
+	$< generate-doc _output/share/man/man1 \
+		--output _output --prefix $(PREFIX)
+
+.PHONY: docsy
+docsy: _output/bin/limactl$(exe)
+	@mkdir -p _output/share/docsy
+	$< generate-doc --type docsy _output/share/docsy \
 		--output _output --prefix $(PREFIX)
```

Comment:
The docsy files are not expected to be installed on the host, so it shouldn't be generated under `_output/share`.

Probably you want to mkdir something like `_output.website` or `website/_output`.

Reply:
This was the main reason for why I never added the markdown or html (or text) rendering to the code.

I did not want redundant copies of man pages installed on the drive, better to use xman (or similar) ?

Reply:
Fixed. Moved to _output/website

Reply:
>  Moved to _output/website

Doesn't seem correct, as this makes the `website` directory  to be installed onto `/usr/local/website`

https://github.com/lima-vm/lima/blob/de1b3ee69561953f71d03cf8fb4b11d72482a350/Makefile#L148-L154

Reply:
My bad missed the install flow. Moved it to website/_output.